src/main/java/ca/corefacility/bioinformatics/irida/ria/web/dto/IridaTemporaryFile.java

```diff
@@ -0,0 +1,33 @@
+package ca.corefacility.bioinformatics.irida.ria.web.dto;
+
+import java.nio.file.Path;
+
+/**
+ * Used as a response for encapsulating a temporary file and it's directory
```

Comment:
Can we update this comment to say something about how this is a temporarily downloaded file from an online service and should be cleaned up after it's done being used?

Reply:
Updated in 117eedc7f06e586f08eed0fca9d520de62ca3c6f

Reply:
Just noticed the package of this file.  Definitely shouldn't be in `dto`.  Probably more in something like `ca/corefacility/bioinformatics/irida/repositories/filesystem`

Reply:
My bad. Been putting all dtos in that directory. Updated in d15b2a07ea240feaf6f39b90428d985fca96efa8